src/load/azext_load/data_plane/load_test_run/custom.py

```diff
@@ -0,0 +1,223 @@
+from azext_load.data_plane.utils.utils import (
+    create_or_update_test_run_body,
+    get_admin_data_plane_client,
+    get_test_run_id,
+    get_testrun_data_plane_client,
+)
+from azure.cli.core.azclierror import ValidationError
+from knack.log import get_logger
+import os, requests
+logger = get_logger(__name__)
+
+
+def create_test_run(
+    cmd,
+    load_test_resource,
+    test_id,
+    display_name=None,
+    existing_test_id=None,
+    description=None,
+    env=None,
+    secrets=None,
+    certificate=None,
+    resource_group_name=None,
+    no_wait=False,
```

Comment:
Is this the az cli convention or something you are creating? Earlier I remembered there was a flag called `--wait` and this seems the opposite

Reply:
its the AZ CLI convention. In the below doc, there is support for `--no-wait` and `--defer` but does not mention anything like `--wait`, but we can support `--wait` ourselves. Its just a Boolean flag so we can add that if it is easier to use for the user

https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands

Reply:
Got it